### PR TITLE
Update yubico-yubikey-piv-manager from 1.4.2f to 1.4.2h

### PR DESCRIPTION
**Pull request by methridge**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

File: Casks/yubico-yubikey-piv-manager.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   pkg "yubikey-piv-manager-#{version}-mac.pkg"
 
   uninstall signal:  ['TERM', 'YubiKey PIV Manager'],
-            pkgutil: 'com.yubico.pkg.YubiKeyPIVManager'
+            pkgutil: ['com.yubico.pkg.YubiKeyPIVManager', 'YubiKey PIV Manager']
```
Comment (miccal) [inline Casks/yubico-yubikey-piv-manager.rb:13]: Please change this to:
```
pkgutil:  [
            'com.yubico.pkg.YubiKeyPIVManager',
            'YubiKey PIV Manager',
          ]
```

**Comment by core-code** [comment]
is the 'com.yubico.pkg.YubiKeyPIVManager' package uninstall no longer needed?

**Comment by miccal** [comment]
@ore-code if it was still needed it would have not passed the CI tests.

**Comment by core-code** [comment]
true :)

**Pull request merged**